client/app/cloud/project/compute/loadbalancer/configure/compute-loadbalancer-configure.less

```diff
@@ -0,0 +1,31 @@
+#compute-loadbalancer-configure {
+    .oui-label {
+        display:inline;
+        padding-top: 8px;
+        margin-right: 15px;
+    }
+    .oui-field {
+        flex-direction: row;
+        flex:none;
+    }
+    h3 {
+        margin-bottom:20px;
```

Comment:
attributes (h3, h2, table) affect the responsiveness of the angular app and should be let by default. I think default behaviour should be fine.
Also, verify spacing for semicolon it should be { argument: value }